src/widget/form/settings/avform.cpp

```diff
@@ -54,6 +54,7 @@ AVForm::AVForm() :
     connect(bodyUI->outDevCombobox, qcbxIndexChangedStr, this, &AVForm::onOutDevChanged);
     connect(bodyUI->videoDevCombobox, qcbxIndexChangedInt, this, &AVForm::onVideoDevChanged);
     connect(bodyUI->videoModescomboBox, qcbxIndexChangedInt, this, &AVForm::onVideoModesIndexChanged);
+    connect(bodyUI->FPSSpinbox, SIGNAL (valueChanged (int)), this, SLOT (onVideoFPSChanged (int)));
```

Comment:
We try to use the new connect syntax from http://doc.qt.io/qt-5/signalsandslots.html